### ISSUE TITLE
Package 'libjpeg62-turbo' has no installation candidate

### ISSUE THREAD

**Issue by wipfli** [maplibre/maplibre-native#1832]
When following the [linux install guidelines](https://github.com/maplibre/maplibre-native/blob/main/platform/linux/README.md) in github codespaces, I get the following error:

```
@wipfli ➜ /workspaces/maplibre-gl-native (harffbuzzFreetype) $ sudo apt install libcurl4-openssl-dev libglfw3-dev libuv1-dev libpng-dev libicu-dev libjpeg62-turbo libwebp-dev

Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libjpeg62-turbo is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libjpeg62-turbo' has no installation candidate
```

**Comment by wipfli** [comment]
And later when trying to compile, I get the following error:

```
@wipfli ➜ /workspaces/maplibre-gl-native (harffbuzzFreetype) $ cmake -S . -B build -G Ninja -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10

CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  CMake 3.19 or higher is required.  You are running version 3.16.3


-- Configuring incomplete, errors occurred!
```

**Comment by ntadej** [comment]
Yes, we bumped minimal CMake version to 3.19. For Ubuntu/Debian you can follow instructions on https://apt.kitware.com or just download CMake binaries manually.

**Comment by ntadej** [comment]
For `libjpeg`, either it's `libjpeg-turbo8-dev` or `libjpeg62-dev`, not 100% sure.

**Comment by louwers** [comment]
We can also include instructions for Ubuntu, but the instructions are for Debian.

**Comment by ntadej** [comment]
Hmm, it's interesting that Ubuntu and Debian have different package names...

**Comment by louwers** [comment]
We can use a custom image for codespaces instead: https://aka.ms/configure-codespace

**Comment by acalcutt** [comment]
In the originally documentation I wrote for debian 11 it noted that I was pulling in the ubuntu version of libjpeg-turbo8

https://github.com/maplibre/maplibre-native/pull/1211/files#diff-978b94a5a8d60e980b1af9814007656a55b78d442e8d276edc102f3e1b8faaaaL17-L24
and
https://maptiler-tileserver.readthedocs.io/en/latest/installation.html#from-source

This was because I could not find a version in debian that worked with the binaries compiled on ubuntu.

Eventually I think someone did tell me there was a package on debian, but i can't seem to find that right now.
